Assets/MRTK/Providers/Oculus/XRSDK/MRTK-Quest/Editor/OculusXRSDKHandtrackingConfigurationChecker.cs

```diff
@@ -228,11 +228,9 @@ static void UpdateCSC()
             List<string> warningNumbers = new List<string>();
 
             // List of new warning numbers to add to the csc file
+            // Empty as of Oculus Integration 39.0 https://developer.oculus.com/downloads/package/unity-integration/39.0
```

Comment:
Interesting...do we need this List and any code path that uses it at all then? As a follow-up, do we need to stay compatible with previous versions of the Integration Package?

Reply:
The code path is in the Oculus Integration Package itself, so not much we can do there.

Oculus doesn't really highlight each package release, so I don't think backwards compatibility should be the highest priority considering their release patterns.